### PR TITLE
Add notebook for creating (and exploring) logos

### LINKED ISSUE #171: ENH: python-graphblas could use a logo!
Any volunteers?  Please respect copywrite licenses.

### PR DESCRIPTION
**Pull request by eriknw**
Aim is to close #171.

I don't trust my eye for artistic or design work, so I tried to make it possible (and hopefully not too difficult) to experiment with both the color palette and logos.

I want to hear feedback! :)

### CODE REVIEW

**Comment by eriknw** [comment]
Oh good, it looks like the images persisted, so it's possible to view the notebook in this PR to see everything. For example: https://github.com/python-graphblas/python-graphblas/blob/f2dee979916a8068109f20a14921f0627bfd0e7f/notebooks/logos_and_colors.ipynb

**Comment by SultanOrazbayev** [comment]
I like variant 1. :)

**Comment by eriknw** [comment]
Thanks @SultanOrazbayev! I could go with variant 1 :)

I'm curious what these look like with the name logo added beside or below.

Also, note that the logos don't currently render correctly on Android devices (b/c they don't have Courier New font).

**Comment by SultanOrazbayev** [comment]
> Also, note that the logos don't currently render correctly on Android devices (b/c they don't have Courier New font).

Whoa, I thought font availability problem was a thing of the past.

**Comment by eriknw** [comment]
If you're wondering where the other secondary colors come from (celadon, eggplant, and dutch white), it's b/c I like the way they look here:
https://coolors.co/visualizer/208bb4-f87a2d-b2c9ab-e8ddb5-463239

**Comment by eriknw** [comment]
Here's variant 1:
logo0
logo1
logo1b
It was recommended that logos be simple, including perhaps not having the white border in the logo directly above this line (i.e., the third logo above). Also, a logo needs to be versatile and usable in many contexts. Let's try making some lines and nodes slightly larger:
logo2
logo3
Better? For me, the white rings here look even more obtrusive than before:
logo4
Now let's consider removing the gap between nodes and edges:
logo5
logo6
Thoughts?

**Comment by eriknw** [comment]
Reversed colors:
logo7
logo8
I recommend zooming out to 20% to 25% to see the logos when smaller.

**Comment by SultanOrazbayev** [comment]
The G is not as recognizable when zoomed-out, but I think that is fine (and expected) at small scale.

**Comment by eriknw** [comment]
One more iteration of simplification! These only use two colors and are the same regardless of light or dark. I also made some things slightly thicker to show up better when displayed as a small favicon. Left/middle/right use slightly different colors, because slightly lighter colors show up better for me when small.

2color_light0 2color_light0b 2color_light0c

2color_dark0 2color_dark0b 2color_dark0c

2color_light1 2color_light1b 2color_light1c

2color_dark1 2color_dark1b 2color_dark1c

The right-most logos use this color palette:
https://coolors.co/85ffc7-ff8552-6d213c-409dc1-39393a (this can also show color-blindness)
https://coolors.co/visualizer/85ffc7-ff8552-6d213c-409dc1-39393a

I'm leaning towards orange nodes and blue lines, which comes across as stronger/bolder for me. This also makes us more similar to NetworkX logo and less similar to NumFOCUS logo.

Compare to NumFOCUS logo:

NF-circle-brackets

and NetworkX logo:

networkx_logo

To recap: the top-right is my leading contender as the logo:

2color_light0c 2color_dark0c

Any final tweaks to try? CC @jim22k

(note: I _did not_ "borrow" any color from any other logo--these colors are our own--but blue/orange was considered so as to be similar to networkx; also, our brackets looks different than the NumFOCUS brackets)

**Comment by eriknw** [comment]
Notebook added, and logo added to repo:

python-graphblas-logo

**Comment by jim22k** [comment]
That works for me.

**Comment by eriknw** [comment]
Merging! Thanks for all the feedback y'all (here and through other channels). This was pretty fun!

I just whipped up the vertical and horizontal versions of the logo (image + text). We can always review and make changes if there's a reason to.
